Strategic-Initiatives.md

```diff
@@ -0,0 +1,39 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  The goal of the TSC is to have a `champion` for each of these
```

Comment:
Minor nit: I think we should avoid using backticks for things that aren't (code) keywords. Perhaps use italics, bold, or quotes instead (here and elsewhere in this document).

Reply:
....Or use nothing. The word *champion* does not need the scare quotes here or any other typographic addition.

Reply:
Ok changed to remove backticks.